addons/cookoff/functions/fnc_cookOff.sqf

```diff
@@ -69,8 +69,8 @@ if (_smokeDelayEnabled) then {
             _vehicle setVariable [QGVAR(isCookingOff), false, true];
             [_pfh] call CBA_fnc_removePerFrameHandler;
 
-            if (_detonateAfterCookoff) then {
-                _vehicle setDamage 1;
+            if (GVAR(destroyVehicleAfterCookoff) || _detonateAfterCookoff) then {
```

Comment:
How about this ... ?
`{_detonateAfterCookoff}`

Granted, I dunno the language well enough to know if it matters.

Reply:
No I was wrong, I changed my review. You're good with `setDamage [1, _detonateAfterCookoff]`